packages/angular/projects/clr-angular/src/progress/progress-bars/progress-bar.ts

```diff
@@ -135,11 +128,12 @@ export class ClrProgressBar {
     this._flashDanger = isBooleanAttributeSet(value);
   }
 
-  // Aria Live
-  /** @deprecated since 3.0, remove in 4.0 */
-  @Input('clrAssertive') assertive: boolean;
-  /** @deprecated since 3.0, remove in 4.0 */
-  @Input('clrOff') off: boolean;
+  /**
+   * Display aria-live only when there is value and it's not 0 or equal to the max value
+   */
+  displayStringValue() {
```

Comment:
This is for the optional value text after the progress bar, right? Why would it be hidden if it was 0 or max?

Reply:
Yes is for showing the optional value after the progress bar .. but I don't have an idea why it is written this way?! 

I'm trying to figure it out. But it seems that the check is "pointless". I think that there is a need for the `ngIf` but the check has to be something else. 

```ts
// show value on the right only when the class label is set to true
displayStringValue() { return this._labeled }
```

Reply:
It seems that this was something that no one notice .. thanks for catching it